make/ocm.mk

```diff
@@ -88,9 +100,24 @@ ocm/cluster/delete:
 	${OCM} delete /api/clusters_mgmt/v1/clusters/$(OCM_CLUSTER_ID)
 
 .PHONY: ocm/cluster.json
-ocm/cluster.json: export OCM_CLUSTER_REGION := "eu-west-1"
+ocm/cluster.json: export OCM_CLUSTER_REGION := eu-west-1
 ocm/cluster.json:
 	@mkdir -p ocm
-	sed "s/OCM_CLUSTER_NAME/$(OCM_CLUSTER_NAME)/g" templates/ocm-cluster/cluster-template.json | \
-	sed "s/OCM_CLUSTER_REGION/$(OCM_CLUSTER_REGION)/g" | \
-	sed "s/OCM_CLUSTER_EXPIRATION_TIMESTAMP/$(OCM_CLUSTER_EXPIRATION_TIMESTAMP)/g" > ocm/cluster.json
+	@jq '\
+	.expiration_timestamp = "$(OCM_CLUSTER_EXPIRATION_TIMESTAMP)" | \
+	.name = "$(OCM_CLUSTER_NAME)" | \
+	.region.id = "$(OCM_CLUSTER_REGION)"' < templates/ocm-cluster/cluster-template.json > ocm/cluster.json
+	@if [ $(BYOC) = true ]; then\
+	  jq '\
+	  .byoc = true | \
+	  .aws.access_key_id = "$(ACCESS_KEY)" | \
+	  .aws.secret_access_key = "$(SECRET_KEY)" | \
+	  .aws.account_id = "$(AWS_ACCOUNT_ID)"' < ocm/cluster.json > ocm/cluster.json.tmp \
+	  && mv ocm/cluster.json.tmp ocm/cluster.json;\
+	fi
+	@cat ocm/cluster.json
+
+.PHONY: ocm/aws/create_access_key
+ocm/aws/create_access_key:
+	@mkdir -p ocm
+	@aws iam create-access-key --user-name osdCcsAdmin | jq -r .AccessKey | tee ocm/aws.json
```

Comment:
Can we add the aws tooling to the opesshift ci Dockerfile or create a follow on Jira for it.

Reply:
sure, I think I can do it as a part of this PR

Reply:
update: discussed and decided TBD as a part of [this jira ticket](https://issues.redhat.com/browse/INTLY-4892)